settings.gradle

```diff
@@ -24,6 +24,7 @@ include 'core'
 include 'data'
 include 'flink'
 include 'mr'
+include 'mr-runtime'
```

Comment:
This name matches the module its based on, another option would be `hive-runtime`.

Reply:
I think we should use `iceberg-hive-runtime`. Eventually, we will probably rename iceberg-mr to iceberg-hive. We also need to rename iceberg-hive to iceberg-hive-metastore.

Reply:
Sure, I'll rename this now. As for the iceberg-hive-metastore rename, I'm happy to do that too, should that be in a separate PR or as part of this one?

Reply:
That would be great, thank you! I think a separate PR.

Reply:
Cool, I'll get that up in the next day or two. Thanks for merging this one.

Reply:
No problem! I think with this one and filter pushdown in, we're about ready for a release with Hive read support! Would be good to get projection pushdown in, but I'm not sure how difficult that will be.

Reply:
Agreed, @cmathiesen is looking into projection pushdown as well as some documentation for the basic Hive read support.